govcd/edgegateway.go

```diff
@@ -131,11 +132,81 @@ func (eGW *EdgeGateway) AddDhcpPool(network *types.OrgVDCNetwork, dhcppool []int
 
 }
 
+// Temporary fix for existing Remove functions - until ID support
```

Comment:
Could you add a TODO listing what should we do when the ID support comes?
And perhaps explain what "ID support" means

Reply:
Update comments